packages/webpack/src/default-es5-settings.js

```diff
@@ -0,0 +1,41 @@
+// const path = require('path');
+// const HtmlWebpackPlugin = require('html-webpack-plugin');
+// const merge = require('webpack-merge');
+// const defaultSettings = require('./default-settings');
+
+// const defaultOptions = {
+//   index: './index.html',
+// };
+
+// eslint-disable-next-line
+module.exports = userOptions => {
+  return {
+    entry: [
+      require.resolve('@webcomponents/webcomponentsjs/webcomponents-bundle.js'),
+      require.resolve('@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'),
+    ],
+    module: {
+      rules: [
+        {
+          test: /\.m?js$/,
```

Comment:
do we want to include the import.meta loader by default as well?

Reply:
I'd say so. It's a web standard.

Reply:
I'd like to do some profiling before concluding that it's slow :)

Reply:
@daKmoR do you agree to pick this up later with profiling?

Reply:
sure